src/Validation/Validation.php

```diff
@@ -521,9 +521,11 @@ public static function localizedTime($check, $type = 'datetime', $format = null)
 
     /**
      * Boolean validation, determines if value passed is a boolean integer or true/false.
+     * To check if a value is set specifically to false or True see `inList()`.
```

Comment:
Makes me wonder if inList fits:
```bash
$ php -a
php > $vals = [0, 1, '0', '1', true, false];
php > foreach($vals as $val) { echo strval($val); }
01011
```
... but should be `010110`, e.g. `strval(false)` doesn't return `0` but rather `''`.


Maybe rather add true/false as a param to boolean() to enforce it to be falsey/truthy? (0, '0', false, 1, '1', true)?